src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -468,11 +468,46 @@ function parseBuildGradleFile(gradleFile: string) {
     throw new Error(`Unable to find gradle file "${buildGradlePath}".`);
```

Comment:
I would put the check `if (fileDoesNotExistOrIsDirectory(buildGradlePath)) {` in `parseBuildGradleFile` instead, because I feel existence of this file shouldn't affect whether we try parsing the gradle properties.